Explorer/Assets/Scripts/Global/Dynamic/GlobalWorld.cs

```diff
@@ -39,6 +41,9 @@ public void Dispose()
         {
             destroyCancellationSource.Cancel();
             worldSystems.Dispose();
+#if UNITY_EDITOR
+            SystemGroupSnapshot.Instance.Unregister(GlobalWorld.WORLD_NAME, worldSystems);
```

Comment:
Instead of wrapping every usage with a directive you could use `Conditional` on the method itself